### DISCUSSION TITLE
How do I delete a duplicate .html from GitHub Pages

### DISCUSSION THREAD

**Discussion by DaynaSommer** [community/community#57188] [New to GitHub] [answered]
How do I delete an unnecessary .html page in GitHub Pages? I can edit it but I cannot figure out how to delete it. Thanks!

![Screen Shot 2023-06-04 at 6 39 16 PM](https://github.com/community/community/assets/119884135/97c817c8-b070-46e3-9038-7d64f1ceb058)

**Comment by radar-blip** [comment, marked as answer]
Once you click on the file, you should have this option on the right side.
Click on the 3 little dots (menu).

![Screenshot from 2023-06-04 21-43-12](https://github.com/community/community/assets/39196767/84a3825b-bb9d-4a32-861a-22d338250929)

**Reply by DaynaSommer** [reply]
Thank you! Initially I did not see the delete because I needed to scroll down.

**Reply by radar-blip** [reply]
You're welcome. Glad it worked for you!
